packages/network/test/index.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { beforeAll, afterAll, afterEach, expect, test, assert } from 'vitest'
+
+import { setupServer } from 'msw/node'
+import { rest } from 'msw'
```

Comment:
Do we need this?

In general a better pattern is to accept a mock replacement in options

e.g.

```ts
buildFetch(middlewares, { fetch: myMockFetch })
```

Reply:
This isn't really testing the network stack though right? The idea of making a real call is ensuring we are testing the stack e2e without worrying about the mock being incomplete.

Reply:
oh i hadn't paid enough attention to understand what `msw` was doing and I got hung up on the word "mock".

This is great 👍